core/dispatcher/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -232,6 +259,7 @@ class Invoker(
                 case None => {
                     activationCounter.next() // this is the global invoker counter
                     incrementUserActivationCounter(tran.msg.subject)
+                    tran.releaseResource(FailedActivation(transid))
```

Comment:
This is not a failed activation path but the path taken for the first one to record the activation completed.